src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzerResultsTableModel.java

```diff
@@ -262,6 +263,19 @@ public String getType() {
         public Map<String, Object> getCustomStates() {
             return customStates;
         }
+        /** 
+         * Represents the 'FuzzResultTableEntry' as a string with all values separated by a comma. Is called by
+         * the Export button's actionListener in 'HttpFuzzResultsContentPanel'. 
+         * 
+         * A HttpFuzzerReflectionDetectorStateHighlighter is used to reveal true value of the customState field
+         */
+        public String toString() {

```

Comment:
The method should be annotated with Override annotation, but relying on the toString method to convert to a format is not the best. This conversion might also lead to problems since some of the values might already have commas.
Consider using a library (for example, [Apache Commons CSV](https://commons.apache.org/proper/commons-csv/)) to do the conversion.
Not all available fields are being exported (for example, URL, method, size request header/body.)